front/app/modules/commercial/content_builder/admin/components/CraftComponents/TwoColumn/index.tsx

```diff
@@ -20,29 +17,60 @@ import Container from '../Container';
 import { FormattedMessage } from 'utils/cl-intl';
 import messages from '../../../messages';
 
-const TwoColumn: UserComponent = ({ columnLayout }) => {
-  const isLargeTablet = useBreakpoint('largeTablet');
+const StyledBox = styled(Box)`
+  min-height: 40px;
+  width: 100%;
+  gap: 16px;
+  display: grid;
+
+  ${media.smallerThanMaxTablet`
+    grid-template-columns: 1fr;
+  `}
+
+  ${media.biggerThanMaxTablet`
+  &.LayoutEven {
```

Comment:
Use a prop instead of classnames like so: https://styled-components.com/docs/basics#adapting-based-on-props